client/app/components/onboardingGuide/GuideListingPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import scss from './styles.scss';
+
+const GuideListingPage = (props) => {
+  const handleClick = function handleClick(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  };
+
+  return (
+    <div className="container">
+      <a className={scss.backLink}
+        onClick={(e) => handleClick(e, '')}
+        href={props.initialPath}
+      >
+        ‹ {props.t('back_to_todo')}
+      </a>
+
+      <h2 className={scss.title} >{props.t('title')}</h2>
+
+      <p className={scss.description} >
+        {props.t('description')}
+      </p>
+
+      <div className={scss.sloganImageContainer} >
+        <img src={props.pageData.info_image}
+          className={scss.sloganImage}
+          alt={props.t('info_image_alt')}
+        />
+      </div>
+
+      <div className={scss.infoTextContainer} >
+        <div className={scss.infoTextIcon} >
+          <i className="ss-info"></i>
```

Comment:
D'oh. This escaped my todo list. 
We can't use ss-info directly. We should use 'information' which is then mapped in Rails side to ss-pika or font-awesome - but the question is that: I'm not sure does it do that on our setup.